src/commands/createFunctionApp/createCreateFunctionAppComponents.ts

```diff
@@ -69,7 +69,8 @@ export async function createCreateFunctionAppComponents(context: ICreateFunction
 
     if (!wizardContext.advancedCreation) {
         LocationListStep.addStep(wizardContext, promptSteps);
-        wizardContext.useFlexConsumptionPlan = true;
+        // if the user is deploying to a container app, do not use a flex consumption plan
+        wizardContext.useFlexConsumptionPlan = true && !context.dockerfilePath;
```

Comment:
Nit: I think the `true` is unnecessary here

Reply:
It doesn't, but I wanted to keep it so it's more explicitly clear that it _should_ be set to true in all cases except where there's a dockerfilePath.

Leaving it as just `context.dockerfilePath` makes it read to me like it's something dockerfile related as in, because we don't have a dockerfile, we need to default to this versus like, this is the normal default, but having a dockerfile will screw it up, so we need to make sure that we don't have one.

Not sure if that distinction makes sense or not.